packages/nuxt3/src/pages/runtime/composables.ts

```diff
@@ -14,7 +14,7 @@ export interface PageMeta {
   [key: string]: any
   pageTransition?: boolean | TransitionProps
   layoutTransition?: boolean | TransitionProps
-  key?: string | ((route: RouteLocationNormalizedLoaded) => string)
+  key?: boolean | string | ((route: RouteLocationNormalizedLoaded) => string)
```

Comment:
@danielroe Is this right? 

```suggestion
  key?: false | string | ((route: RouteLocationNormalizedLoaded) => string)
```